test/CMakeLists.txt

```diff
@@ -35,8 +37,18 @@ if(NOT DLAF_MPI_PRESET STREQUAL _DLAF_MPI_PRESET)
           FORCE
     )
     set(MPIEXEC_NUMCORES "" CACHE STRING "Number of cores available for each MPI rank." FORCE)
+    if(DLAF_TEST_THREAD_BINDING_ENABLED)
+      message(WARNING "Disabling pika binding")
+      set(DLAF_TEST_THREAD_BINDING_ENABLED FALSE CACHE BOOL "" FORCE)
+    endif()
```

Comment:
I'm sorry if this was already discussed somewhere: is the idea that if `preset == plain-mpi` then that essentially implies a debug environment where one never threads to be bound? If I'm on a machine with enough cores but still using `plain-mpi` I probably still want to bind threads because tests will likely still finish faster (even if I may not care about absolute performance).

So my question is: would it be appropriate to set the _default_ of `DLAF_TEST_THREAD_BINDING_ENABLED` to off if the preset is `plain-mpi` but still allow it to be changed back to on?

Reply:
This case is quite complicated as in general MPI(*) doesn't setup correct rank binding and therefore the different ranks (as our test spawn multiple rank per node) would anyway be bound to the same cores.
So the binding in that case would be beneficial only to 1 RANK tests.

(*) Even if MPI takes care of the rank binding, IIRC currently there is no way to set the thread/rank flag in the `plain-mpi` case and the binding would be wrong.

Reply:
> in general

is the key point in my opinion, because MPI can do this (sometimes). But my question is also not very important in the bigger scheme. I was just curious as to why it's forced rather than only "preferred", but I'm not expecting changes here!